src/components/volunteer-form/VolunteerCreatedPage.tsx

```diff
@@ -90,17 +93,21 @@ export const VolunteerCreatedPage: React.FC<VolunteerCreatedPageProps> = ({
                         </Stack>
                     </Box>
                 </Center>
+            ) : successful === "pending" ? (
+                <Spinner />
```

Comment:
idem, let's use Loading here.
```suggestion
                <Loading />
```